libc/src/langinfo/nl_langinfo.cpp

```diff
@@ -0,0 +1,20 @@
+//===-- Implementation of nl_langinfo -------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "src/langinfo/nl_langinfo.h"
+
+#include "src/__support/common.h"
+
+namespace LIBC_NAMESPACE_DECL {
+
+LLVM_LIBC_FUNCTION(char *, nl_langinfo, (nl_item)) {
+  static char EMPTY[] = "";
```

Comment:
if these aren't implemented we should provide an error to the user. I'd recommend setting `errno` to `ENOSYS`.